pha-admin/src/main/java/com/store/pharmacy/common/serviceImpl/MessageServiceImpl.java

```diff
@@ -4,6 +4,7 @@
 import com.store.pharmacy.common.model.MessageOutput;
 import com.store.pharmacy.common.repository.MessageRepository;
 import com.store.pharmacy.common.service.MessageService;
+import com.store.pharmacy.exception.DataNotFoundException;
```

Comment:
A package name should not be named serviceImpl due to naming convention violation. Also check other classes.
Suggestion:

<img width="180" alt="image" src="https://user-images.githubusercontent.com/86360944/137623412-b0d2583a-92a7-4679-8c8a-4e9f1926b859.png">